registration/tests/serializers/test_family.py

```diff
@@ -23,41 +28,89 @@ def setUp(self):
             address="1 Django Boulevard",
             preferred_comms="Shark Tune",
         )
-
         self.family_without_children = Family.objects.create(
             parent=self.parent1,
             email="justkeepswimming@ocean.com",
             cell_number="123456789",
             address="42 Wallaby Way",
             preferred_comms="Whale Song",
         )
-
-        self.family_without_parent = Family.objects.create(
+        self.family_status = Family.objects.create(
+            parent=self.parent3,
             email="justkeepswimming@ocean.com",
             cell_number="123456789",
             address="1 Test Ave",
             preferred_comms="Dolphin Whistle",
         )
-
         self.child1 = Student.objects.create(
             first_name="Albus",
             last_name="Whale",
             role=Student.CHILD,
             family=self.family,
         )
-
         self.child2 = Student.objects.create(
             first_name="Lily",
             last_name="Whale",
             role=Student.CHILD,
             family=self.family,
         )
-
         self.child3 = Student.objects.create(
             first_name="Harry",
             last_name="Tuna",
             role=Student.CHILD,
-            family=self.family_without_parent,
+            family=self.family_status,
```

Comment:
nit: children aren't needed for the enrolment status test, so we can remove this!